lib/card.rb

```diff
@@ -14,11 +14,9 @@ class Card < ActiveRecord::Base
 
   has_many :references_from, :class_name => :Reference, :foreign_key => :referee_id
   has_many :references_to,   :class_name => :Reference, :foreign_key => :referer_id
-  has_many :acts, :order => :id
-  has_many :actions, :order => :id, :conditions=>{:draft => [nil,false]}
-  has_many :drafts, :order=>:id, :conditions=>{:draft=>true}, :class_name=> :Action
-
-  cache_attributes 'name', 'type_id' # review - still worth it in Rails 3?
+  has_many :acts, -> { order :id }
+  has_many :actions, -> { where( :draft=>[nil,false]).order :id }
+  has_many :drafts, { :class_name=> :Action }, -> { where( :draft=>true ).order :id }
```

Comment:
It's a new lambda shortcut as of ruby 1.9.1 (http://ruby.about.com/od/newinruby191/qt/LambdaOperator.htm)

The notion has been that the upgrade to Rails 4 would imply moving Wagn's minimal ruby requirement to 1.9.3 (with a preference for 2).

I wouldn't swear that it's impossible to support older rails versions, but it would be a significant challenge I don't see a ton of value there, do you?